src/rng.cpp

```diff
@@ -88,3 +88,9 @@ double rng_normal( double lo, double hi )
     double val = std::normal_distribution<double>( ( hi + lo ) / 2, range )( eng );
     return std::max( std::min( val, hi ), lo );
 }
+
+double normal_roll( double mean, double stddev )
+{
+    static std::default_random_engine eng;
```

Comment:
The random engine has to be randomly initialized (sorry for the tautology), otherwise it would return same sequence of numbers all the time.